config/routes.rb

```diff
@@ -263,11 +263,9 @@
     end
   end
 
-  resources :audits, :only => [:index] do
-    collection do
-      get 'auto_complete_search'
-    end
-  end
+  match '/audits/auto_complete_search' => 'audits#auto_complete_search', :via => [:get]
+  match '/audits/load' => 'audits#index', :via => [:get]
```

Comment:
maybe use the same you did with the statistics pr? (so we dont need to introduce a new url?)